src/Service/MarketoMaApiClientInterface.php

```diff
@@ -100,4 +100,17 @@ public function syncLead(Lead $lead, $key = 'email', $options = []);
    */
   public function deleteLead($leads, $args = array());
 
+  /**
+   * Associate a known lead with activity recorded by a cookie.
+   *
+   * @param $leadId
+   *   The id of the lead whose activity should be associated with the cookie.
+   * @param $cookie
+   *   The browser cookie value to associate with the lead.
+   * @param array $args
+   *
+   * @return mixed
+   */
+  public function associateLead($leadId, $cookie, $args = array());
```

Comment:
Not seeing that args is a valid parameter here: http://developers.marketo.com/rest-api/endpoint-reference/lead-database-endpoint-reference/#!/Leads/associateLeadUsingPOST